jenkins/vulnerability-scan/whitesource-scan.jenkinsfile

```diff
@@ -1,7 +1,7 @@
 pipeline {
     agent none
     options {
-        timeout(time: 5, unit: 'HOURS')
+        timeout(time: 7, unit: 'HOURS')
```

Comment:
just curious why this is changed

Reply:
@tianleh We used to run scan on main branches and this PR is to add 1.3 branch along with main branch. I assume the time should be doubled. I changed it to 7 hours so far and will need to increase it if it's not enough.